aten/src/ATen/native/native_functions.yaml

```diff
@@ -8904,6 +8904,20 @@
   dispatch:
     DefaultBackend: det
 
+- func: linalg_lstsq(Tensor self, Tensor b, float? cond=None, str? driver_name=None) -> (Tensor solution, Tensor rank, Tensor singular_values)
```

Comment:
Let's kwarg "driver_name" and rename it to "driver."

If we were designing PyTorch today we would have been much more aggressive about making things kwarg-only, just like the new proposed tensor API from Quansight does. In this particular case, I'd also like to recommend that "driver" be kwarg-only because if we want to add other SciPy arguments like, like "check_finite", we don't want to worry about trying to insert a positional argument before another one.